src/main/java/com/grum/geocalc/Point.java

```diff
@@ -56,6 +78,20 @@ public Point(Coordinate latitude, Coordinate longitude) {
     public double getLatitude() {
         return latitude;
     }
+    
+    public Coordinate getLatitudeCoordinate() {
+        return DegreeCoordinate.build(latitude);
+    }
+
+    public void setLatitude(double latitude) {
+        this.latitude = latitude;
```

Comment:
also the whole point of not having setter was to make those Point objects immutable.